content/en/docs/concepts/scheduling/configure-scheduler.md

```diff
@@ -0,0 +1,223 @@
+---
+title: How to Configure Your Kubernetes Scheduler
+reviewers:
+- wgliang
+content_template: templates/concept
+weight: 30
+toc_hide: false
+---
+
+{{% capture overview %}}
+
+This is the third article on how to use and extend the Kubernetes scheduler series. The series of articles are listed below:
+
+* [How does the Kubernetes scheduler work?](docs/concepts/scheduling/how-scheduler-works/)
+* [How to customize and extend the Kubernetes scheduler?](/docs/concepts/scheduling/customization-and-extension/)
+* How to configure your Kubernetes scheduler?
+* [Choose scheduling algorithms and strategies for your cluster.](TODO)
+* [Learn the kube-scheduler's design and implementation from source code.](TODO)
+
+This page explains how to configure Kubernetes scheduler.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Scheduler
+
+The [Kubernetes scheduler](/docs/reference/command-line-tools-reference/kube-scheduler/) is a main component of Kubernetes. It is responsible for selecting the best node for the pod based on a series of predefined scheduler policies. Kubenetes provides the default scheduler and policies to place the pod on the appropriate node in a Kubernetes cluster.
+
+## Configure the Policies
+
+The Kubernetes scheduler plugin frameworks provides the user with the ability to:
+
+1. Customize the default scheduler policies. 
+2. Extend the scheduler.
+3. Write the new scheduler(s) to either run alongside the default scheduler or replace the default scheduler.
+
+For more information, see the article [Customization and Extension ](/docs/concepts/scheduling/customization-and-extension/).
+
+The default scheduler policies fit most use cases. Users can fine-tune the scheduler behavior without modifying the default scheduler by specifying the customized-policy file when the kube-scheduler starts with the `--policy-config-file` option.
+
+Given below is an example of a typical policy file:
+
+```json
```

Comment:
This example will help the reader to understand how the JSON-based policy file is structured since the previous paragraph mentioned the policy file. 

Probably we don't have to always keep it updated and the user can easily author the policy files by following our example and adding/deleting the values Kubernetes supports/will support. If we're able to automatically generated one and guarantee it updated alway , it will be the best scenario and very very helpful and we can put the link here instead. :-)